azure-functions/acbs-function/src/functions/acbs-amend-facility.js

```diff
@@ -138,23 +138,28 @@ df.app.orchestration('acbs-amend-facility', function* amendFacility(context) {
       amendments,
     });
 
+    /**
+     * TODO: DTFS2-7581
+     * Below have been disabled for release 2.4.0.
+     * Enable below two alongside with investor and fixed fee records.
+     */
     // 3. SOF: Facility Covenant Record (FCR)
-    const facilityCovenantRecord = yield context.df.callSubOrchestrator('acbs-amend-facility-covenant-record', {
-      facilityIdentifier,
-      amendments,
-    });
+    // const facilityCovenantRecord = yield context.df.callSubOrchestrator('acbs-amend-facility-covenant-record', {
```

Comment:
should this be going into main?

Reply:
Yep as this is blocked until we can test this so bringing `main` to same state as `prod`.

Reply:
It reads odd that it's disabled for release `2.4.0` but we're now merging `2.5.1`